### DISCUSSION TITLE
5/15 (월) 케로 - 통합테스트 분리하기

### DISCUSSION THREAD

**Discussion by jyeost** [woowacourse-study/2023-presentation-study#28] [Q&A]
[통합 테스트 분리하기](https://edony.notion.site/c49fcaf2c0d84be98997080fe74ab0f6?pvs=4)

**Comment by kwonyj1022** [comment]
외부 db를 사용했을 때 초기화 방법 3가지 중에 어떤 방법이 더 좋은가요?

**Comment by sh111-coder** [comment]
* `@Sql`에서 excutionPhase 속성의 기본 설정은 뭔가요?

**Comment by yeonkkk** [comment]
main과 test의 resources에 동일한 이름을 가진 sql 파일이 있다면 `@Sql` scripts에서는 어떤 파일을 우선해서 찾는지 궁금합니다!
